serverless/knative_eventing/serverless-event-sinks.adoc

```diff
@@ -0,0 +1,21 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-event-sinks"]
+= Event sinks
+include::modules/common-attributes.adoc[]
+:context: serverless-event-sinks
+
+toc::[]
+
+A sink is an xref:../../serverless/knative_eventing/serverless-understanding-eventing.adoc#serverless-understanding-eventing-objects[_Addressable_ resource] that can receive incoming events from other resources. Knative services, channels, and brokers are all examples of sinks.
```

Comment:
A sink is defined as an _Addressable resource_ here, but the link goes to _Addressable objects_. Should these be consistent?

Reply:
yeah I'll change them all to resources perhaps / CRs